apps/dashboard/src/data/editor/selectors.js

```diff
@@ -2,7 +2,26 @@
  * External dependencies
  */
 import { serialize, parse } from '@wordpress/blocks';
-import { get, isEmpty, map, pickBy, slice, some } from 'lodash';
+import {
+	cloneDeep,
+	filter,
+	get,
+	isEmpty,
+	isEqual,
+	map,
+	pickBy,
+	slice,
+	some,
+	tap,
+} from 'lodash';
+import { createSelectorCreator, defaultMemoize } from 'reselect';
+
+// Creates a 'createSelector' instance using lodash 'isEqual'
+// function as the comparison method
+const createDeepEqualSelector = createSelectorCreator(
+	defaultMemoize,
+	isEqual
```

Comment:
Are you sure you need deep equality here? The objects are nested, yes, but since everything comes from Redux stores, we should be able to rely on regular `createSelector`.

Reply:
As we discussed, I removed the deep equality here and changed the styles-resolver logic a bit to prevent unnecessary settings updates.